### DISCUSSION TITLE
We are unable to complete your signup

### DISCUSSION THREAD

**Discussion by magickdeveloper** [vercel/community#3270] [Help] [answered]
### Summary

I try to login but i get that message.
When i tried to verify my account this happen:
![image](https://github.com/vercel/community/assets/127857899/3999fa85-6507-41dc-97c1-4f3be9afcfd4)

### Steps to Reproduce

Register, verify and login

**Comment by amyegan** [comment, marked as answer]
Hey @magickdeveloper.  If you tried to sign up with a GitHub account, you may have run into a problem we are currently investigating: https://www.vercel-status.com/incidents/590pcktngq7z

If you ran into problems signing up with email or a different git repo host, please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username for the account.

**Reply by rovarkamil** [reply]
how long does it usually take for them to answer?

**Reply by amyegan** [reply]
Tickets are processed by the order in which they are received, meaning it can take some time for the team to respond to all of the earlier cases that were opened before they reach yours. 

For more information about response times, please visit https://vercel.com/guides/vercel-support-queue-time

**Reply by votruongsanh** [reply]
Has anyone received a reply from Vercel yet? thanks

**Reply by anitacheung83** [reply]
It should takes 3 - 4 weeks for the review. I am waiting for mine too  :(

**Reply by rovarkamil** [reply]
it's been 4 months for me and no answer, maybe it differ for country regions.

**Comment by najmus-dev** [comment]
same issue for me
